pkg/web/apps/apps_http_handler.go

```diff
@@ -64,19 +64,57 @@ func (a *httpHandler) UpdateApp(c echo.Context) error {
 
 }
 
-// GetAppByID returns apps by id as JSON from the AppService
-func (a *httpHandler) GetAppByID(c echo.Context) error {
+//UpdateApp returns a app updated with the ID in JSON format from the AppService
+func (a *httpHandler) UpdateAppVersions(c echo.Context) error {
+	// Validations
+	id := c.Param("id")
+	if !helpers.IsValidUUID(id) {
+		return httperrors.BadRequest(c, "Invalid id supplied")
+	}
+
+	versions := []models.Version{}
+	errV := json.NewDecoder(c.Request().Body).Decode(&versions)
+
+	// check if the data sent is in the correct format
+	if errV != nil {
+		return httperrors.BadRequest(c, "Invalid data")
+	}
 
+	// Check if versions were sent all the body is empty
+	if len(versions) == 0 {
+		return httperrors.BadRequest(c, "No version(s) was sent.")
+	}
+
+	// Call service
+	errUpdate := a.Service.UpdateAppVersions(versions)
+	if errUpdate != nil {
+		return httperrors.GetHTTPResponseFromErr(c, errUpdate)
+	}
+
+	return c.JSON(http.StatusOK, "")
+}
+
+//UpdateApp returns a app updated with the ID in JSON format from the AppService
+func (a *httpHandler) DisableAllAppVersionsByAppID(c echo.Context) error {
 	id := c.Param("id")
-	if !helper.IsValidUUID(id) {
+	if !helpers.IsValidUUID(id) {
 		return httperrors.BadRequest(c, "Invalid id supplied")
 	}
 
-	apps, err := a.Service.GetAppByID(id)
+	ver := models.Version{}
+	errV := json.NewDecoder(c.Request().Body).Decode(&ver)
+
+	// check if the data sent is in the correct format
+	if errV != nil {
+		return httperrors.BadRequest(c, "Invalid data")
+	}
+
+	err := a.Service.DisableAllAppVersionsByAppID(id, ver.DisabledMessage)
```

Comment:
Quick question on how to verify this endpoint do I pass in an object with 

```
{
	DisabledMessage:      "Please contact an administrator",
}
```

Reply:
Hi @austincunningham the Version struct has the JSON object mapped as `disabledMessage`.

Reply:
Thank you for doing this comment I note that it was missing an adjust. Also, note that all steps are now in the first comment. Please, pull the PR branch locally. Really tks for your help on it.

Reply:
:+1:

Reply:
can't seem to verify
![image](https://user-images.githubusercontent.com/16667688/53624516-7d340480-3bf8-11e9-8247-a381c052677b.png)

Reply:
@austincunningham you receive Not Found which happened because the URL is wrong. The server answered that the URL was not found. My tip is you go in the router and cut and past the URL.

Reply:
wrong url , can verify now